fairlearn/datasets/_common.py

```diff
@@ -0,0 +1,4 @@
+class FairnessWarning(UserWarning):
```

Comment:
I think `fairlearn/utils/_common.py` would be a better place for this one?

Reply:
Agreed.

Reply:
How about `fairlearn.exceptions` (to mirror `sklearn.exceptions`)? Also--how specific do we want to scope this warning? If we only expect to use it for data sets, we could use a more specific name.

Reply:
Working on an iteration now. I've renamed it to `UnfairDataWarning` and moved the warning to `fairlearn.exceptions`.